mmcls/models/backbones/efficientnet_v2.py

```diff
@@ -0,0 +1,395 @@
+    pad_h, pad_w = get_same_padding(ih, k[0], s[0], d[0]), get_same_padding(
+        iw, k[1], s[1], d[1])
+    if pad_h > 0 or pad_w > 0:
+        x = F.pad(
+            x,
+            [pad_w // 2, pad_w - pad_w // 2, pad_h // 2, pad_h - pad_h // 2],
+            value=value)
+    return x
+
+
+def conv2d_same(x,
+                weight: torch.Tensor,
+                bias: Optional[torch.Tensor] = None,
+                stride: Tuple[int, int] = (1, 1),
+                padding: Tuple[int, int] = (0, 0),
+                dilation: Tuple[int, int] = (1, 1),
+                groups: int = 1):
+    x = pad_same(x, weight.shape[-2:], stride, dilation)
+    return F.conv2d(x, weight, bias, stride, (0, 0), dilation, groups)
+
+
+class Conv2dSame(nn.Conv2d):
+    """Tensorflow like 'SAME' convolution wrapper for 2D convolutions."""
+
+    def __init__(self,
+                 in_channels,
+                 out_channels,
+                 kernel_size,
+                 stride=1,
+                 padding=0,
+                 dilation=1,
+                 groups=1,
+                 bias=True):
+        super(Conv2dSame,
+              self).__init__(in_channels, out_channels, kernel_size, stride, 0,
+                             dilation, groups, bias)
+
+    def forward(self, x):
+        return conv2d_same(x, self.weight, self.bias, self.stride,
+                           self.padding, self.dilation, self.groups)
+
+
+class ConvBNAct(nn.Module):
+
+    def __init__(self,
+                 in_planes: int,
+                 out_planes: int,
+                 kernel_size: int = 3,
+                 stride: int = 1,
+                 groups: int = 1,
+                 padding: int = 1,
+                 norm_layer: Optional[Callable[..., nn.Module]] = None,
+                 activation_layer: Optional[Callable[..., nn.Module]] = None):
+        super(ConvBNAct, self).__init__()
+
+        padding = (kernel_size - 1) // 2
+        if norm_layer is None:
+            norm_layer = nn.BatchNorm2d
+        if activation_layer is None:
+            activation_layer = nn.SiLU  # alias Swish
+
+        if stride == 1:
+            self.conv = nn.Conv2d(
+                in_channels=in_planes,
+                out_channels=out_planes,
+                kernel_size=kernel_size,
+                stride=stride,
+                padding=padding,
+                groups=groups,
+                bias=False)
+        elif stride == 2:
+            self.conv = Conv2dSame(
+                in_channels=in_planes,
+                out_channels=out_planes,
+                kernel_size=kernel_size,
+                stride=stride,
+                groups=groups,
+                bias=False)
+
+        self.bn = norm_layer(out_planes)
+        self.act = activation_layer(inplace=True)
+
+    def forward(self, x):
+        result = self.conv(x)
+        result = self.bn(result)
+        result = self.act(result)
+
+        return result
+
+
+class SqueezeExcite(nn.Module):
```

Comment:
the class can be replaced by https://github.com/open-mmlab/mmclassification/blob/458ac4c89ae18ae49750b6e66a30f1a08720b77e/mmcls/models/utils/se_layer.py#L10

Reply:
Difficult, the first ConvModule in SELayer need SiLU active function， but ConvModule  doesn`t support SiLU active function.

Reply:
I know silu is same as swish , so i will change my code style as mmcls style.